xonsh/pretty.py

```diff
@@ -115,6 +115,9 @@ def pretty(obj, verbose=False, max_width=79, newline='\n', max_seq_length=MAX_SE
     """
     Pretty print the object's representation.
     """
+    if hasattr(obj, 'xonsh_display'):
+        obj.xonsh_display()
```

Comment:
I think this needs to `return obj.xonsh_display()`

Reply:
Yes, I copied my example and I had it printing from the xonsh_display().